src/main/java/com/joshdev/deathmanager/commands/RestoreInventory.kt

```diff
@@ -0,0 +1,141 @@
+/* Licensed under GNU General Public License v3.0 */
+package com.joshdev.deathmanager.commands
+
+import com.joshdev.deathmanager.DeathManager
+import com.joshdev.deathmanager.libs.Serialization
+import net.kyori.adventure.text.Component
+import net.kyori.adventure.text.format.NamedTextColor
+import net.kyori.adventure.text.format.TextDecoration
+import org.bukkit.command.Command
+import org.bukkit.command.CommandExecutor
+import org.bukkit.command.CommandSender
+import org.bukkit.entity.Player
+
+class RestoreInventory : CommandExecutor {
+    override fun onCommand(sender: CommandSender, command: Command, label: String, args: Array<out String>?): Boolean {
```

Comment:
Should use `cloud` framework for this, would be so much cleaner.

Reply:
This might take a little time for me to resolve, will update as development progresses.